integr8ly/integreatly-operator
# MGDAPI-3462 Automate b08b test: Verify User RHSSO Permissions

# Issue link

https://issues.redhat.com/browse/MGDAPI-3462

# What

Automate test b08b to validate that both regular user and dedicated-admins user is allowed to create a Realm in User SSO.

# Verification steps

- Provision an OSD cluster

- Install RHOAM

- Once RHOAM is fully installed, Run test  `TEST="Verify users can create a Realm in User SSO" make test/e2e/single`

- Once the test has ran, follow steps to verify:

  - Log into API Management SSO via `incognito window.`
  - Log into the Testing IDP for each user: 
  - `Username: test-user01 Password: Password1`
  - `Username: customer-admin01 Password: Password1`
- A new Realm should now be created for each user with a random Hexadecimal naming format.
